src/rtc_pl031.rs

```diff
@@ -166,8 +178,8 @@ impl<EV: RTCEvents> RTC<EV> {
             // Counts up from 1 on reset at 1Hz (emulated).
             counter: Instant::now(),
 
-            // The load register is initialized to zero.
-            lr: 0,
+            // The load register is initialized to UNIX epoch time.
+            lr: get_epoch_time(),
```

Comment:
This comment applies to `test_control_register` as well.

Reply:
I would leave the example as is because the driver can still set the LR value (and UNIX Epoch is  a value that makes sense I think), and the example correctly checks that a value set in LR, can then be read from the register.

Reply:
I've updated `test_control_register`.

Reply:
Looks like with the new changes my comment doesn't make sense anymore. [This one](https://github.com/rust-vmm/vm-superio/compare/c0491934e3ff2ff3e3fa3efc22b4254fb23ed5ee..f66c624522ba0acdb5a7e5bddfec4ba7f8205c84#diff-20e2d11876ae9564e0e163379ffa125952680cfc6b71daf9644f52a892a23b58R178) should be updated though.

Reply:
Done.